docker-compose.yml

```diff
@@ -122,6 +122,45 @@ services:
     labels:
       logs: "promtail"
 
+### Web Client
+  hyperswitch-web:
+    ports:
+      - "9050:9050"
+      - "9060:9060"
+      - "5252:5252"
+    build:
+      context: ./docker
+      dockerfile: web.Dockerfile
+    networks:
+      - router_net
+    env_file:
+      - ./.env
+    environment:
+      - HYPERSWITCH_PUBLISHABLE_KEY=$HYPERSWITCH_PUBLISHABLE_KEY
+      - HYPERSWITCH_SECRET_KEY=$HYPERSWITCH_SECRET_KEY
+      - HYPERSWITCH_SERVER_URL=${HYPERSWITCH_SERVER_URL:-http://hyperswitch-server:8080}
+      - HYPERSWITCH_CLIENT_URL=${HYPERSWITCH_CLIENT_URL:-http://localhost:9050}
+      - SELF_SERVER_URL=${SELF_SERVER_URL:-http://localhost:5252}
+    depends_on:
+      hyperswitch-server:
+        condition: service_started
+
+  ### Control Center
+  hyperswitch-control-center:
+    image: juspaydotin/hyperswitch-control-center
+    ports:
+      - "9000:9000"
+    networks:
+      - router_net
```

Comment:
this might not be needed since there isn't any inter-service communication happening here...
since all communication happens on the host machine and is initiated by browser.

Reply:
![image](https://github.com/juspay/hyperswitch/assets/67261499/d5d1ec3a-6c1f-4fa5-8d1a-5e2ce3d6520e)
@lsampras have tested the flow added one fix for the same.